### PR TITLE
Consensus dashboard account list card

### PR DESCRIPTION
**Pull request by buberdds**
Screenshot from 2024-01-18 11-56-26

### CODE REVIEW

File: src/oasis-nexus/api.ts [hunk 1]
```diff
@@ -777,3 +784,42 @@ export const useGetRuntimeEvmTokensAddressHolders: typeof generated.useGetRuntim
...
+      ...options?.request,
+      transformResponse: [
+        ...arrayify(axios.defaults.transformResponse),
+        (data: generated.AccountList, headers, status) => {
+          if (status !== 200) return data
+          return {
+            ...data,
+            accounts: data.accounts.map(account => {
+              // TODO: remove when API returns total, looking at query filters we store that data in Nexus
+              const total = BigInt(account.available) + BigInt(account.escrow) + BigInt(account.debonding)
```
Comment (lukaw3d) [inline src/oasis-nexus/api.ts:806]: This calculation is wrong. API does not return the correct fields, and debonding is named misleadingly. History of wrong math: https://github.com/oasisprotocol/nexus/issues/84 and https://github.com/oasisprotocol/oasis-wallet-web/pull/927

https://nexus.oasis.io/v1/consensus/accounts?limit=1
> "available":"0","debonding":"0","escrow":"0"

single account API returns correct fields
https://nexus.oasis.io/v1/consensus/accounts/oasis1qz78ap0456g2rk7j6rmtvasc9v2kjhz2s58qgj90
> "available":"21454060793","debonding":"0","debonding_delegations_balance":"0","delegations_balance":"103544825467","escrow":"0"

File: src/app/components/AccountSizeBadge/index.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  fontSize: '10px',
+  fontWeight: 700,
+}))
+
+type AccountSizeBadgeProps = {
+  size: string
+}
+
+export const AccountSizeBadge: FC<AccountSizeBadgeProps> = ({ size }) => {
+  return <StyledBox>{size}</StyledBox>
```
Comment (buberdds) [inline src/app/components/AccountSizeBadge/index.tsx:28]: Add tooltip

File: src/app/components/AccountList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    { align: TableCellAlign.Right, key: 'total', content: t('account.totalBalance') },
+  ]
+  const tableRows = accounts?.map((account, index) => ({
+    key: account.address,
+    data: [
+      {
+        align: TableCellAlign.Center,
+        // TODO: replace index when rank is implemented in the API
+        content: <div>{index + 1}</div>,
+        key: 'rank',
```
Comment (buberdds) [inline src/app/components/AccountList/index.tsx:36]: After discussion with Don we can kill rank column

File: src/locales/en/translation.json [hunk 4]
```diff
@@ -1,16 +1,21 @@
 {
   "appName": "Oasis Explorer",
   "account": {
+    "available": "Available",
     "cantLoadDetails": "Unfortunately we couldn't load the account details at this time. Please try again later.",
+    "debonding": "Debonding",
     "emptyTokenList": "This account holds no {{spec}} {{description}}.",
     "emptyTransactionList": "There are no transactions on record for this account.",
     "emptyTokenTransferList": "There are no token transfers on record for this account.",
+    "escrow": "Staked",
```
[the remark this answers is not in the document]
Comment (buberdds) [reply src/locales/en/translation.json]: ah sorry was looking at prop names and was comparing them with wallet

File: src/oasis-nexus/api.ts (hunk 1 above)
Comment (buberdds) [reply src/oasis-nexus/api.ts]: I will leave TODOs in this case.

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**